### PR TITLE
Update Channel Sequence docs

### PR DESCRIPTION
**Pull request by alicenstar**
Relates to this story: https://app.shortcut.com/replicated/story/101398/show-the-channel-release-sequence-on-the-channel-history-page

Preview: https://deploy-preview-2217--replicated-docs-upgrade.netlify.app/vendor/releases-about#channel-sequences

### CODE REVIEW

**Comment by alicenstar** [comment]
@paigecalvert Let me know if you think I should do a more close up screenshot, not really sure on the preferred style there.

File: docs/vendor/releases-about.md [hunk 1]
```diff
@@ -260,4 +262,4 @@ You can do the following tasks on the **Draft** page:
 
 - Select the Config custom resource to preview how your application's Config page will look to your customers. The **Config preview** pane only appears when you select that file. For more information, see [About the Configuration Screen](config-screen-about).
 
-- Select the Application custom resource to preview how your application icon will look in the admin console. The **Application icon preview** only appears when you select that file. For more information, see [Customizing the Application Icon](admin-console-customize-app-icon).
\ No newline at end of file
+- Select the Application custom resource to preview how your application icon will look in the admin console. The **Application icon preview** only appears when you select that file. For more information, see [Customizing the Application Icon](admin-console-customize-app-icon).
```
Comment (alicenstar) [inline docs/vendor/releases-about.md:265]: I'm not really sure why this is showing changes... I didn't touch anything here and it looks the same.

Comment (paigecalvert) [reply docs/vendor/releases-about.md:269]: Sometimes it'll highlight the last line of the topic in the Files changed...not sure why

**Review by paigecalvert — APPROVED** [review]
thanks @alicenstar ! The screenshot looks good to me, I just edited it to add a little red box around the channel sequence numbers so that they stand out

**Pull request merged**